apps/dashboard/src/components/primitives/field-editor/variable-plugin/variable-theme.ts

```diff
@@ -0,0 +1,61 @@
+import { EditorView } from '@uiw/react-codemirror';
+import { VARIABLE_PILL_CLASS } from './constants';
+
+export const variablePillTheme = EditorView.baseTheme({
+  [`.${VARIABLE_PILL_CLASS}`]: {
+    backgroundColor: 'hsl(var(--bg-weak))',
+    color: 'hsl(var(--text-sub))',
+    border: '1px solid hsl(var(--stroke-soft))',
+    borderRadius: '10px',
+    padding: '2px 6px 2px 6px',
+    fontFamily: 'inherit',
+    display: 'inline-flex',
+    alignItems: 'center',
+    height: '16px',
+    lineHeight: '16px',
+    fontSize: '12px',
+    cursor: 'pointer',
+    position: 'relative',
+    marginRight: '0px',
+    top: '2px',
+  },
+  '.cm-variable-pill::before': {
+    content: '""',
+    left: '4px',
+    width: '12px',
+    minWidth: '12px',
+    height: '12px',
+    marginRight: '3px',
+    backgroundColor: 'hsl(var(--feature-base))',
+    maskImage: `url("/images/code.svg")`,
```

Comment:
If not mistaken we can do `content: url("/images/code.svg");`. Is the maskImage required for anything else? If not, let's simplify it.